docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md

```diff
@@ -2,7 +2,7 @@
 
 The [api:SS_List] class is designed to return a flat list of records.
 These lists can get quite long, and hard to present on a single list.
-[Pagination](/templates/how_tos/pagination) is one way to solve this problem,
+[Pagination](/en/3.0/howto/pagination/) is one way to solve this problem,
```

Comment:
If it's "/howto/pagination/", then the whole url is "http://doc.silverstripe.org/howto/pagination/", which will give users message "We're sorry…
The page you are looking for does not exist, or might have moved."
The "Pagination" part is only working in version 3.0, that's why I added the special version.
Please refer to http://doc.silverstripe.org/en/developer_guides/model/how_tos/grouping_dataobject_sets/